fava/util/date.py

```diff
@@ -35,27 +35,72 @@
     '|'.join(list(modifiers.keys())),
     '|'.join(all_months + ['month', 'year'])))
 
-
-def get_next_interval(date, interval):
+rel_date_re = re.compile(
+    # optional space, to support "10d ago" and "10 days ago"
+    r'(\d+)[ ]?'
+    # w, week weeks
+    # d, day, days
+    # q, quarter, quaters
+    # y, year, years
+    r'(d(?=ays?)?|w(?=eeks?)?|q(?=uarters?)?|y(?=ears?)?)[^ ]*'
+    r'[ ](ago|later)')
```

Comment:
"later" is not the correct word here. It's not an antonym of "ago", I think we should rather use "from now".